pkg/audit/manager.go

```diff
@@ -96,13 +96,13 @@ type Manager struct {
 
 // StatusViolation represents each violation under status.
 type StatusViolation struct {
-	Group             string `json:"group"`
-	Version           string `json:"version"`
-	Kind              string `json:"kind"`
-	Name              string `json:"name"`
-	Namespace         string `json:"namespace,omitempty"`
-	Message           string `json:"message"`
-	EnforcementAction string `json:"enforcementAction"`
+	Group             string   `json:"group"`
+	Version           string   `json:"version"`
+	Kind              string   `json:"kind"`
+	Name              string   `json:"name"`
+	Namespace         string   `json:"namespace,omitempty"`
+	Message           string   `json:"message"`
+	EnforcementAction []string `json:"enforcementAction"`
```

Comment:
I don't think we can change the schema here without breaking backwards compatibility.